exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/translate/protobuf.py

```diff
@@ -190,16 +190,24 @@ def _extract_tags(self, span: Span) -> Sequence[model_pb2.KeyValue]:
                 if key_value:
                     translated.append(key_value)
 
-        code = _get_long_key_value(
-            "status.code", span.status.status_code.value
-        )
-        message = _get_string_key_value(
-            "status.message", span.status.description
-        )
-        kind = _get_string_key_value(
-            "span.kind", OTLP_JAEGER_SPAN_KIND[span.kind]
+        status = span.status
+        if status.status_code is not StatusCode.UNSET:
+            translated.append(
+                _get_string_key_value(
+                    "otel.status_code", status.status_code.name
+                )
+            )
+            if status.description is not None:
```

Comment:
should this be outside of the status_code if? is it possible for someone to set a description while leaving the status code unset?

Reply:
>is it possible for someone to set a description while leaving the status code unset?

They can do it with `.set_status(Status(StatusCode.UNSET, "desc"))` if they choose to. We report `otel.status_code` and `otel.status_description` only when status code is OK or ERROR. This should be inside.

Reply:
[Link](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk_exporters/jaeger.md#status) to specs for this. Did I misunderstand it?

Reply:
Looks like your implementation is correct!